<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DuMediaPlayerPreferencesDialog</class>
 <widget class="QDialog" name="DuMediaPlayerPreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferencias</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Idioma</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/lang.png</normaloff>:/lang.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Estilo</string>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/style.png</normaloff>:/style.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page_1">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QListWidget" name="langListWidget">
           <item>
            <property name="text">
             <string>Español</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/mx.png</normaloff>:/mx.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Inglés</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/us.png</normaloff>:/us.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QListWidget" name="styleListWidget">
           <item>
            <property name="text">
             <string>Windows Vista</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/windowsVista.png</normaloff>:/windowsVista.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Fusion</string>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/fusion.png</normaloff>:/fusion.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="accept">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/accept.png</normaloff>:/accept.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
